wrappers/python/nntile/layer/__init__.py

```diff
@@ -32,4 +33,5 @@
 __all__ = ('Act', 'Add', 'AddSlice', 'Attention', 'AttentionSingleHead',
         'BaseLayer', 'BatchNorm2d', 'Conv2d', 'Embedding', 'FlashAttention',
         'GAP', 'LayerNorm', 'Linear', 'LlamaAttention', 'GPT2Attention',
+        'BertSelfAttention',
```

Comment:
no `BertAttention`?

Reply:
BertAttention is in the module subpackage since it consists of more than one layer inside